TypeInjections/Tool/Tool.fs

```diff
@@ -1,53 +1,97 @@
-﻿
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
 namespace Tool
 
+open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
 
 module Tool =
-    
+
+    //NOTE 'location' refers to either the github url OR path to repo on local computer
+    let checkoutCommit (SHA: string) (location: string) (outputPath: string) : unit =
+        Utils.log $"path to repo is {outputPath}"
```

Comment:
CI is giving warnings here, looks like a simple fix?

Reply:
I can give approve if this is blocking you @tschuerl

Reply:
Yeah the warning is about the variable `SHA` being in all capital letters, I can just change the variable name